src/state/contexts/RegistrationState.tsx

```diff
@@ -8,13 +9,16 @@ export type RegistrationState = {
   domain?: string;
   leaseDuration: number;
   tier: number;
+  antContract?: ANTContract;
+  // following 6 keys replaced by single antContract object
```

Comment:
can we remove them?

Reply:
ah looks like we're using them, let's create a task to update them

Reply:
[PE-3264](https://ardrive.atlassian.net/browse/PE-3264)

Reply:
Actually addressing this in the atomic registration PR  at the moment